scripts/verify_verification_trigger_contract.py

```
@@ -0,0 +1,365 @@
+    lines = ci_text.splitlines()
+    start = None
+    job_indent = None
+
+    for idx, line in enumerate(lines):
+        if line.strip() == f"{job_name}:":
+            start = idx
+            job_indent = len(line) - len(line.lstrip(" "))
+            break
+
+    if start is None or job_indent is None:
+        raise ValueError(f"ci.yml missing job '{job_name}'")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)